client/package.json

```diff
@@ -25,4 +25,4 @@
   "jest": {
     "preset": "react-native"
   }
-}
\ No newline at end of file
+}
```

Comment:
Looks like there might be a missing package here - `react-native-login`? `yarn add react-native-login` should clear it up.